### DISCUSSION TITLE
[Docs][Roadmap] When will metrics/logs be "Stable"?

### DISCUSSION THREAD

**Discussion by JoeLoser** [open-telemetry/opentelemetry-cpp#1684] [General]
In https://opentelemetry.io/docs/instrumentation/cpp/, it mentions that metrics and log features are experimental.  Can you share the roadmap/plans/timeline on when we think metrics and logs will become officially _Stable_?  Thanks!

**Comment by esigo** [comment]
Hi @JoeLoser, thanks for reaching out to us.
Please refer to our [milestone](https://github.com/open-telemetry/opentelemetry-cpp/milestones).

**Reply by JoeLoser** [reply]
Hi @esigo — thanks for the quick response.  I took a look at the milestone and that answers my question re metrics.  I interpret GA to mean general availability to the public, i.e. mostly stable — is that a correct understanding?

Is there any update(s) regarding logs, or is that meant to be bundled alongside the metrics milestone(s)? Thanks!

**Reply by esigo** [reply]
- yes, we don't expect big changes for api :)

- currently the focus is on metrics. There is some discussion about logs [here](https://github.com/open-telemetry/opentelemetry-cpp/discussions/1653) . Please note that the specs for logs is still [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/api.md).
